src/styles/utility/_spacing.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import * as variables from '../variables';
+import notes from './_spacing.stories.md';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const SpacingDemo = (props: any) => (
+    <div
+        style={{
+            background: variables.bdlGreenLight50,
+            margin: 10,
+            display: 'inline-block',
+        }}
+        {...props}
+    />
+);
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Box = (props: any) => (
+    <div {...props} style={{ display: 'inline-block', backgroundColor: variables.bdlBoxBlue, color: '#fff' }}>
+        {props.className}
+    </div>
+);
+
+const Spacing = () => (
+    <>
+        <h2>Spacing Utility Classes</h2>
+
+        <SpacingDemo>
+            <Box className="bdl-px-3 bdl-py-4 bdl-m-3" />
+        </SpacingDemo>
+
+        <SpacingDemo>
+            <Box className="bdl-px-5 bdl-py-2 bdl-m-4" />
```

Comment:
So with above changes, this will read as
```jsx
<Box className="bdl-u-paddingTop-5 bdl-u-paddingRight-5 bdl-u-paddingLeft-2 bdl-u-paddingBottom-2  bdl-u-margin-4" />
```